### ISSUE TITLE
Selected item in "relationship" property type highlighted twice

### ISSUE THREAD

**Issue by zaplvlad** [payloadcms/payload#422]
This is just UI issue. Selected item in "relationship" property type highlighted two times and dropdown itself is partially hidden.

## Steps to reproduce:
1. Create collection and "global" with following configuration:

```
import { CollectionConfig } from 'payload/types';

export const links: CollectionConfig = {
  slug: 'links',
  labels: {
    singular: 'Link',
    plural: 'Links',
  },
  admin: {
    useAsTitle: 'name',
  },
  access: {
    read: () => true,
    update: (a) => {
      const user = a.req.user;
      const roles: string[] = user.roles;
      return roles.some((role) => role === 'admin');
    },
  },
  fields: [
    {
      name: 'name',
      type: 'text',
      label: 'Name',
      unique: true,
    },
    {
      name: 'url',
      type: 'text',
      localized: true,
      label: 'Url',
    },
  ],
};
```
```
import { GlobalConfig } from 'payload/types';
import { links } from '../collections/links';

export const globalCollection: GlobalConfig = {
  slug: 'globalCollection',
  label: 'Global Collection',
  fields: [
    {
      name: 'link',
      type: 'relationship',
      relationTo: links.slug,
      hasMany: false,
    },
  ],
};
```

2. ​Add few items to Links collection

![1](https://user-images.githubusercontent.com/43339104/150781236-f8e2a996-1e16-42b5-a7f9-d5efbe2494e2.png)

3. Select one of them in "global" document and save it.

![2](https://user-images.githubusercontent.com/43339104/150781264-7dcb2675-2f2f-4d58-878e-beff3ad1de12.png)

4. Reload page and open dropdown again. You will see that dropdown itself partially hidden and selected item highlighted two times.

![3](https://user-images.githubusercontent.com/43339104/150781284-5efb2f1b-0b56-4baa-98cb-571d453b9bc4.png)

## Expected result
Selected item should be highlighted once. Dropdown visibility fixed.

**Comment by jmikrut** [comment]
Good catch! This one's actually on our radar already and we have a fix coming.

Thank you very much for the report!

**Comment by jmikrut** [comment]
Hey @zaplvlad — this one is fixed and released as beta version `payload@0.14.21-beta.0`.

@oranoran I believe you also mentioned this issue. Updating should see the issue resolved.

Thank you everyone!
